Framework/Router.php

```diff
@@ -51,8 +51,8 @@ private function call($target, $args, $request) {
     }
 
     private function route($request) {
-
         $modulesNames = Configuration::get("modules");
+        $modulesNames = is_array($modulesNames) ? $modulesNames : [$modulesNames];
```

Comment:
Ok, but why?
Modules are defined in config file as an array

Reply:
That's a security. Documentation indicates that Configuration::get() returns a string. In some similar cases in the framework, I encountered functions returning a string for real instead of an array when there was only one element.
But you're right, in this case I doubt it to be useful.

Reply:
Get doesn t always return a string, it returns result of parse ini.
For elements like:

modules[] it returns an array.
This is a doc error